rosidl_typesupport_connext_cpp/package.xml

```diff
@@ -11,15 +11,15 @@
   <buildtool_depend>connext_cmake_module</buildtool_depend>
   <buildtool_depend>rosidl_cmake</buildtool_depend>
   <buildtool_depend>rosidl_generator_c</buildtool_depend>
```

Comment:
Why is this still referring to the generator?

Reply:
https://github.com/ros2/rosidl_typesupport_connext/pull/49/commits/10197e939bac8025991db110642145638f8d9182